examples/bbcsounds_example.py

```diff
@@ -0,0 +1,47 @@
+"""
+Example on how to use the BBC iPlayer Controller
+
+"""
+# pylint: disable=invalid-name
+
+import logging
+from time import sleep
+import sys
+
+import pychromecast
+from pychromecast.controllers.bbcsounds import BbcSoundsController
+
+# Change to the name of your Chromecast
+CAST_NAME = "Lounge Video"
+
+# Media ID can be found in the URL
+# e.g. https://www.bbc.co.uk/sounds/live:bbc_radio_one
```

Comment:
This is a great explanation, add it in pychromecast/controllers/bbcsounds.py as well